composer.json

```diff
@@ -38,13 +38,13 @@
         "rector/rector-symfony": "^0.11.42",
         "sebastian/diff": "^4.0.4",
         "ssch/typo3-rector": "^0.11.31",
-        "symfony/console": "^5.3|^6.0",
+        "symfony/console": "6.0 as 5.4",
         "symfony/contracts": "^2.5",
-        "symfony/dependency-injection": "^5.3|^6.0",
+        "symfony/dependency-injection": "6.0 as 5.4",
         "symfony/deprecation-contracts": "^2.4",
-        "symfony/finder": "^5.3|^6.0",
-        "symfony/process": "^5.3|^6.0",
-        "symfony/yaml": "^5.3|^6.0",
+        "symfony/finder": "6.0 as 5.4",
+        "symfony/process": "6.0 as 5.4",
+        "symfony/yaml": "6.0 as 5.4",
```

Comment:
The last current blocker is php-cs-fixer https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6095